packages/error-boundary-fallback/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@opensrp/error-boundary-fallback",
+  "version": "0.0.0",
+  "bugs": {
+    "url": "https://github.com/opensrp/web/issues"
+  },
+  "scripts": {
+    "test": "jest --coverage --verbose --color",
+    "transpile": "babel src -d dist --root-mode upward --extensions '.ts,.tsx'  --ignore '**/*.test.ts,**/*.test.tsx,**/tests,**/__tests__'",
+    "tsc": "tsc",
+    "copy": "copyfiles -u 1 \"./src/**/*.{css,html}\" \"./dist/\"",
+    "build": "yarn run tsc && yarn run transpile && yarn run copy"
```

Comment:
Looks like some commands are missing as per the `@opensrp/template`. Could we have the package.json to match the one in the template?